packages/fxa-content-server/webpack.config.js

```diff
@@ -46,7 +45,7 @@ const webpackConfig = {
   },
 
   resolve: {
-    extensions: ['.ts', '.js','.jsx'],
+    extensions: ['.ts', '.js', '.jsx'],
```

Comment:
extra space

Reply:
That was added automatically, looks like it was missing before.

Reply:
woops, right! 👁